### PR TITLE
feat: typography support tooltip

### PR DESCRIPTION
**Pull request by chenshuai2144**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [x] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |       Typography support tooltip    |
| 🇨🇳 Chinese |       Typography 支持 `tooltip`  配置  |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: components/typography/Base.tsx [hunk 1]
```diff
@@ -421,6 +428,21 @@ class Base extends React.Component<InternalBlockProps, BaseState> {
...
 
+  /**
+   * 根据 props.tooltip 判断要不要展示 tooltip
+   */
+  renderTextNode(textNode: React.ReactNode, ariaLabel?: string) {
+    const { tooltip } = this.props;
+    if (tooltip) {
+      return (
+        <Tooltip title={ariaLabel} {...tooltip}>
+          <span aria-hidden="true">{textNode}</span>
```
Comment (zombieJ) [inline components/typography/Base.tsx:439]: 这个 `aria-hidden` 干啥的？

**Comment by zombieJ** [comment]
https://github.com/ant-design/ant-design/issues/23188#issuecomment-615876071

当年做 `ellipsis` 的时候就有讨论过要不要 tooltip，后来决定是不需要。你需要它的场景是什么？

File: components/typography/Base.tsx (hunk 1 above)
Comment (chenshuai2144) [reply components/typography/Base.tsx:439]: 看见你写了，就顺便放上去了

**Comment by chenshuai2144** [comment]
image


为了这种场景

**Comment by afc163** [comment]
原生 title 不够用么

**Comment by chenshuai2144** [comment]
体验不是很好，还是 tooltip 比较靠谱一些

textRender倒是也能满足需求，我改成支持 textRender吧

**Comment by afc163** [comment]
这个 dom 结构很简单了，textRender 没什么必要，还不如直接包装。

**Review by 07akioni — APPROVED** [review]

**Comment by afc163** [comment]
按之前的讨论，应该是去掉 copy 按钮的 Tooltip。

**Pull request closed without merging**